test/torch_test.py

```diff
@@ -985,6 +985,30 @@ def test_mpc_mul(self):
         self.mpc_mul(3, 5)
         self.mpc_mul(2 ** 12, 2 ** 12)
 
+    def test_mpc_matmul(self):
+        x = torch.LongTensor([[1, 2], [3, 4]])
+        y = torch.LongTensor([[5, 6], [7, 8]])
+
```

Comment:
Maybe alos add a test where the result contains poisitve and negative values

Reply:
Good call - adding now

Reply:
@LaRiffle - added - passes on my machine

Reply:
👍